src/devices/sonoff.ts

```diff
@@ -467,18 +439,74 @@ const definitions: Definition[] = [
                 .withLabel('Open window detection')
                 .withDescription('Automatically turns off the radiator when local temperature drops by more than 1.5°C in 4.5 minutes.')
                 .withAccess(ea.ALL),
-            e.numeric('frost_protection_temperature', ea.ALL)
-                .withValueMin(4.0)
-                .withValueMax(35.0)
-                .withValueStep(0.5)
-                .withUnit('°C')
-                .withDescription(
-                    'Minimum temperature at which to automatically turn on the radiator, if system mode is off, to prevent pipes freezing.'),
         ],
-        fromZigbee: [fz.thermostat, fz.battery, fzLocal.child_lock, fzLocal.open_window, fzLocal.frost_protection_temperature],
+        fromZigbee: [
+            fz.thermostat,
+            fz.battery,
+            fzLocal.child_lock,
+            fzLocal.open_window,
+        ],
         toZigbee: [
-            tz.thermostat_local_temperature, tz.thermostat_local_temperature_calibration, tz.thermostat_occupied_heating_setpoint,
-            tz.thermostat_system_mode, tz.thermostat_running_state, tzLocal.child_lock, tzLocal.open_window, tzLocal.frost_protection_temperature],
+            tz.thermostat_local_temperature,
+            tz.thermostat_local_temperature_calibration,
+            tz.thermostat_occupied_heating_setpoint,
+            tz.thermostat_system_mode,
+            tz.thermostat_running_state,
+            tzLocal.child_lock,
+            tzLocal.open_window,
+        ],
+        extend: [
+            extend.numeric({
+                name: 'frost_protection_temperature',
+                cluster: 0xFC11,
+                attribute: {id: 0x6002, type: 0x29},
+                description: 'XXX Minimum temperature at which to automatically turn on the radiator, ' +
```

Comment:
I don't understand why you added `XXX` here.

Reply:
Ha ha, good spot. I temporarily put the 'XXX' in there as a sanity check whilst testing to ensure the UI was rendering the modern expose and not the legacy expose. I forgot to remove it when committing last night.😴